wsdl/ver10/device/wsdl/devicemgmt.wsdl

```diff
@@ -2891,6 +3091,46 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 			<wsdl:input message="tds:GetWsdlUrlRequest"/>
 			<wsdl:output message="tds:GetWsdlUrlResponse"/>
 		</wsdl:operation>
+		<wsdl:operation name="GetPasswordComplexityOptions">
+			<wsdl:documentation>This method allows to retrieve all the available parameters and their valid ranges for the password complexity configuration.</wsdl:documentation>
+			<wsdl:input message="tds:GetPasswordComplexityOptionsRequest"/>
+			<wsdl:output message="tds:GetPasswordComplexityOptionsResponse"/>
+		</wsdl:operation>
+		<wsdl:operation name="GetPasswordComplexityConfiguration">
+			<wsdl:documentation>This method allows to get the current password complexity configuration settings.</wsdl:documentation>
+			<wsdl:input message="tds:GetPasswordComplexityConfigurationRequest"/>
+			<wsdl:output message="tds:GetPasswordComplexityConfigurationResponse"/>
+		</wsdl:operation>
+		<wsdl:operation name="SetPasswordComplexityConfiguration">
+			<wsdl:documentation>This method allows to set the password complexity configuration.</wsdl:documentation>
+			<wsdl:input message="tds:SetPasswordComplexityConfigurationRequest"/>
+			<wsdl:output message="tds:SetPasswordComplexityConfigurationResponse"/>
+		</wsdl:operation>
+		<wsdl:operation name="GetPasswordHistoryConfiguration">
+			<wsdl:documentation>This method allows to get the current password history configuration settings.</wsdl:documentation>
+			<wsdl:input message="tds:GetPasswordHistoryConfigurationRequest"/>
+			<wsdl:output message="tds:GetPasswordHistoryConfigurationResponse"/>
+		</wsdl:operation>
+		<wsdl:operation name="SetPasswordHistoryConfiguration">
+			<wsdl:documentation>This method allows to set the password history configuration.</wsdl:documentation>
+			<wsdl:input message="tds:SetPasswordHistoryConfigurationRequest"/>
+			<wsdl:output message="tds:SetPasswordHistoryConfigurationResponse"/>
+		</wsdl:operation>
+		<wsdl:operation name="GetAuthFailureWarningOptions">
+			<wsdl:documentation>This method allows to retrieve all the available parameters and their valid ranges for the auhentication failure warning configuration.</wsdl:documentation>
```

Comment:
I feel this should read ;

This method allows retrieval of all the available parameters and ....